tests/templates/kuttl/opa-authorization/11-opa-rego.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      # Trino Rego rules provided by the Stackable Data Platform
+      kubectl create configmap trino-opa-rules \
+        --namespace=$NAMESPACE \
+        --from-file=trino_rules/actual_permissions.rego \
+        --from-file=trino_rules/requested_permissions.rego \
+        --from-file=trino_rules/util.rego \
+        --from-file=trino_rules/verification.rego
+      kubectl label configmaps trino-opa-rules \
+        opa.stackable.tech/bundle=trino \
```

Comment:
```suggestion
        opa.stackable.tech/bundle=true \
```

Reply:
changed in ed98c309b897a4ae2f098a20ba9c77304b848e2a